infra/build/functions/trial_build.py

```diff
@@ -95,11 +97,27 @@ def _get_production_build_statuses(build_type):
   return results
 
 
+def handle_special_projects(args):
```

Comment:
Rather than shoehorning language names into project names, is it possible to do: `all --language ...` ?

Reply:
Unless you think this interface is totally off, I think it's much better, it's more intuitive and harder to get wrong. Which is important since the feedback loop isn't as tight as when using a real command line tool.